Index: app/src/main/java/is/hi/hbv601g/kosmosinn_mobile/BoardActivity.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package is.hi.hbv601g.kosmosinn_mobile;\r\n\r\nimport android.os.Bundle;\r\nimport android.widget.TextView;\r\n\r\nimport androidx.appcompat.app.AppCompatActivity;\r\nimport androidx.cardview.widget.CardView;\r\nimport androidx.recyclerview.widget.RecyclerView;\r\n\r\nimport com.android.volley.RequestQueue;\r\nimport com.android.volley.toolbox.Volley;\r\n\r\nimport is.hi.hbv601g.kosmosinn_mobile.R;\r\n\r\npublic class BoardActivity extends AppCompatActivity {\r\n    private static final String TAG = \"BoardActivity\";\r\n    private TextView textView;\r\n    private TextView textV;\r\n\r\n    @Override\r\n    protected void onCreate(Bundle savedInstanceState) {\r\n        super.onCreate(savedInstanceState);\r\n        setContentView(R.layout.activity_board);\r\n        //textView = (TextView) findViewById(R.id.textView);\r\n    }\r\n\r\n    RequestQueue queue = Volley.newRequestQueue(this);\r\n    String url = \"http://192.168.0.103:8080/api/topics\";\r\n\r\n}
===================================================================
--- app/src/main/java/is/hi/hbv601g/kosmosinn_mobile/BoardActivity.java	(revision c708684865d8ad65fd2e0fd2c0d0958bb1a05d4f)
+++ app/src/main/java/is/hi/hbv601g/kosmosinn_mobile/BoardActivity.java	(date 1616954277417)
@@ -24,7 +24,4 @@
         //textView = (TextView) findViewById(R.id.textView);
     }
 
-    RequestQueue queue = Volley.newRequestQueue(this);
-    String url = "http://192.168.0.103:8080/api/topics";
-
 }
\ No newline at end of file
